src/wallet/wallet.h

```diff
@@ -986,4 +986,49 @@ class CAccount
     }
 };
 
+class Witnessifier : public boost::static_visitor<bool>
```

Comment:
Sounds like a good refactor for another PR, especially if I get rid of importprivkey commit.